content/application-service-api.md

```diff
@@ -207,6 +207,33 @@ processed the events.
 
 {{% http-api spec="application-service" api="transactions" %}}
 
+##### Pushing EDUs
```

Comment:
I know that the MSC used the term "EDU", but I think it's somewhat misleading. In particular: we do *not* forward `m.direct_to_device` EDUs to ASes (not least because such EDUs contain multiple to-device messages, possibly for other users).

Also: presence/receipts/typing from users on the same server don't arrive via EDUs, so the current wording would imply that the AS would only receive updates from users on other servers, which I don't believe was the intention of the MSC.

Also also: the event format used by the C-S API (and hence, now, by the A-S API) is quite different from that used by EDUs in the S-S API.

Could we, instead of saying "EDU", say "ephemeral data" ?